build.yaml

```diff
@@ -717,6 +707,23 @@ common_cli:
     wget https://raw.githubusercontent.com/AndrewDryga/vagrant-box-osx/master/Vagrantfile
     vagrant up
 
+  install_aws_cli: &install_aws_cli |
+    # installs the AWS python based cli
+    # we use the aws cli to upload the docs files to S3
+    \${PARSE_LOGS}  pip install awscli \${PIP_ADDITIONAL_OPTIONS}
+
+  get_s3_creds_for_bucket_staging_docs: &get_s3_creds_for_bucket_staging_docs |
+    # collects the S3 credentials that allows us to upload files to the S3
+    # staging bucket for our docs
+    . /tmp/s3_staging_docs_clusterhq_com.sh
+
```

Comment:
That comment seems relevant to the code, not just to the review. Please add it as a comment in the yaml file.